caravel/templates/index_continuation.html

```diff
@@ -1 +1,7 @@
-{% for listing in listings %}{% include "listings/inline.html" %}{% endfor %}
\ No newline at end of file
+{% for listing in listings %}
+  {% if view == "List" %}
+    {% include "listings/list.html" %}
```

Comment:
`inline-block` has weird semantics when there's whitespace — but jinja2 lets you remove whitespace around tags by adding dashes (so it'd be `{%- stuff -%}` instead of  `{% stuff %}`).